docs/node.md

```diff
@@ -99,25 +99,30 @@ Node Controller is unable to provision the node for you, i.e. it won't install
 any binary; therefore, to
 join Kubernetes cluster, you as an admin need to make sure proper services are
 running in the node. In the future, we plan to automatically provision some node
-services. In case of no cloud provider, Node Controller simply registers all
-machines from `--machines` flag, any further interactions need to be done manually
-by using `kubectl`. If you are paranoid, leave `--machines` empty and create all
-machines from `kubectl` one by one - the two approaches are equivalent.
-Optionally you can skip cluster-wide node synchronization with
-'--sync_nodes=false' and can use REST api/kubectl cli to add/remove nodes.
-
-Node life-cycle management in the Node Controller is still under development, it
-is supposed to manage the Node Status Specification defined above.
-
-### Manual Node Administration
-
-A Kubernetes administrator typically uses `kubectl` to manage `Node`. Similar
-to Node Controller, `kubectl` command only creates/deletes node representation.
-Note if Kubernetes is running on cloud provider, `kubectl create` a node will
-be refused if Node Controller has already synchronized nodes from cloud provider.
-Admin can choose to make the node unschedulable using `kubectl`. Unscheduling the node
-will not affect any existing pods on the node but it will disable creation of
-any new pods on the node. Node unschedulable example:
+services. 
+
+### Registration of nodes
+
+The preferred pattern for node registration, used by most of the distros, is for the kubelet binary to
+create and update its own node resource.   The kubelet is started with the following options:
+  - `--apiservers=` tells the kubelet the location of the apiserver.
+  - `--kubeconfig` tells kubelet where to find credentials to authenticate itself to the apiserver.  
+  - `--cloud_provider=` tells the kubelet how to talk to a cloud provider to read metadata about itself.
+  - `--register-node` tells the kubelet to create its own node resource.
+
+Currently, any kubelet is authorized to create/modify any node resource, but in practice it only creates/modifies
+its own.  (In the future, we plan to limit authorization to only allow a kubelet to modify its own Node resource.)
+
+#### Manual Node Administration
+
+Buy setting - `--register-node=0` for all kubelets, the kubelet will not create its own Node, and the administrator
+can do this.
+
+Regardless of the setting of `--register-node`, the adminstrator can modify Node resource, such as modifying labels on the Node, or marking it unschedulable..
+
+Making a node unscheduleable will not affect any existing pods on the node but it will disable creation of
+any new pods on the node. This is useful as a preparatory step before a node reboot, etc.
```

Comment:
Will the schedule avoid nodes that are marked unscheduleable? If the answer is yes, then maybe we can reword this a bit. 

```
... on the node but it will prevent new pods from being scheduled on the node.
```